src/tasks/base-search-task.ts

```diff
@@ -0,0 +1,31 @@
+// global
+import { FastifyLoggerInstance } from 'fastify';
+import { Actor, Task, TaskStatus, IndividualResultType, PreHookHandlerType, PostHookHandlerType, DatabaseTransactionHandler } from 'graasp';
+import { Member } from 'graasp';
+// local
+import { SearchService } from '../db-service';
+
+export abstract class BaseSearchTask<R> implements Task<Actor, R> {
```

Comment:
Could you add some additional properties? I think `graasp` has an update version of it (containing input, getResult, etc). (base-task.ts)